packages/gatsby/cache-dir/fast-refresh-overlay/utils.js

```diff
@@ -1,25 +1,17 @@
 import Anser from "anser"
 
-const enterRegex = /^\s$/
-
 export function prettifyStack(errorInformation) {
   let txt
   if (Array.isArray(errorInformation)) {
     txt = errorInformation.join(`\n`)
   } else {
     txt = errorInformation
   }
-  const generated = Anser.ansiToJson(txt, {
+  return Anser.ansiToJson(txt, {
```

Comment:
I was seeing weird behavior with removing the first line for all different errors. The underlying error format must have improved so this isn't necessary anymore

Reply:
🤞 This doesn't give me much confidence 😛 Maybe it happened because of https://github.com/gatsbyjs/gatsby/pull/30801

Reply:
> I was seeing weird behavior with removing the first line for all different errors.

Should rephrase that: The indentation for the now first line was slightly incorrect as the previously existing new line wasn't there anymore.

The PR you linked might have caused this, yeah